src/sidebar/components/user-menu.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const bridgeEvents = require('../../shared/bridge-events');
+const { isThirdPartyUser } = require('../util/account-id');
+const serviceConfig = require('../service-config');
+const { withServices } = require('../util/service-context');
+
+const Menu = require('./menu');
+const MenuSection = require('./menu-section');
+const MenuItem = require('./menu-item');
+
+/**
+ * A menu with user and account links.
+ *
+ * This menu will contain different items depending on service configuration,
+ * context and whether the user is first- or third-party.
+ */
+function UserMenu({ auth, bridge, onLogout, serviceUrl, settings }) {
+  const isThirdParty = isThirdPartyUser(auth.userid, settings.authDomain);
+  const service = serviceConfig(settings);
+
+  const serviceSupports = feature => service && !!service[feature];
+
+  const isSelectableProfile =
```

Comment:
These two lines have been converted to Booleans instead of functions and the logic has been changed (being in the sidebar is no longer evaluated, but being a first-party user is).